### ISSUE TITLE
[Scaffolding] Rollup Bundle Analyzer

### ISSUE THREAD

**Issue by aress31** [open-wc/open-wc#1704]
I use `open-wc` to quickly scafflod and configure a new web application projects. I am not entirely sure that `open-wc` was designed to achieve this purpose but since the `PWA Starter Kit` points to this repository and the availability of a `scaffold a new application` option, I think that yes, please correct me if I wrong.

I am not super familiar with `Rollup` but I would like to be able to inspect and analyse my application bundle size and chunks - when code splitting is enabled - should not there be a `NPM script` available out of the box to launch the analyzer?

**Comment by bennypowers** [comment]
You can add [rollup-plugin-analyzer](https://github.com/doesdev/rollup-plugin-analyzer) to your config. it's [relatively straightforward](https://open-wc.org/building/building-rollup.html#extending-the-rollup-config) to add your own plugins.

**Comment by aress31** [comment]
@bennypowers thanks for directing me to this plugin, I think that the documentation could benefit from a few explanation lines on how to set it up or a direct link to this repository.

**Comment by LarsDenBakker** [comment]
@aress31 Are you up for adding some docs for this?

**Comment by aress31** [comment]
@LarsDenBakker, absolutely! Could you please direct me to the right file to edit? I will add references to `rollup-bundle-analyzer` and `rollup-plugin-analyzer` to help with optimising the build size.

**Comment by LarsDenBakker** [comment]
It should be in the readme of the `building-rollup` package.
